### PR TITLE
Paintroid 398 No error message shown when Catrobat stickers are not available

### PR DESCRIPTION
**Pull request by illo1311**
Added a function where it is checked if the device is connected to the internet and if not a toast is shown when you want to access the online stickers. 
https://jira.catrob.at/secure/RapidBoard.jspa?rapidView=138&view=detail&selectedIssue=PAINTROID-398

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.kt [hunk 1]
```diff
@@ -944,6 +947,9 @@ open class MainActivityPresenter(
 
     override fun importStickersClicked() {
         navigator.showCatroidMediaGallery()
+        if (!checkForInternet(context)) {
+            Toast.makeText(context, "Cannot access stickers at the moment. Please check your internet connection, or try again later.", Toast.LENGTH_LONG).show()
```
Comment (StanglTobias) [inline Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.kt:951]: Please move the text to the resource file

**Review by StanglTobias — DISMISSED** [review]
did you also check if it is possible for the `MediaGalleryWebViewClient` to fail even if we are connected to the network?

**Comment by pratyaksh1610** [comment]
@StanglTobias I would like to work on this issue, pls assign this issue to me if it is inactive .
.
.
Thankyou
Pratyaksh

**Review by ThorstenBandel — APPROVED** [review]
LGTM

**Comment by juliajulie95** [comment]
Thank you for the implementation!
When testing I saw that the message is too long for my phone.
Maybe we could shorten iut a bit. @ThorstenBandel ?
Screenshot_20221228_152623

**Review by juliajulie95 — APPROVED** [review]

**Pull request merged**